UnityProject/Assets/Scripts/Items/Attributes.cs

```diff
@@ -188,6 +188,11 @@ public RightClickableResult GenerateRightClickOptions()
 
 	public void ServerSetArticleName(string newName)
 	{
+		if (gameObject.TryGetComponent<Stackable>(out var stack))
+		{
+			newName = newName + $" ({stack.Amount})";
+		}
+		newName = newName.Replace("[item]", $"{initialName}");
```

Comment:
Whats this for?

Reply:
for setting the name from the inspector
It is meant for the default item names for stackable items that do not have a special stack name yet.
so in the inspector it will be "a stack of [item]" and in game it will be "a stack of Plasma Ore" and so on.